requirements/base.txt

```diff
@@ -1,8 +1,9 @@
 # Requirements for app run
 
-git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==1.1.1
-django-statici18n==1.8.0
-edx-i18n-tools==0.4.7
-Mako==1.0.7
-bleach==2.1.4
-html-xblock~=0.1.1
+git+https://github.com/edx/xblock-utils.git@2.1.1#egg=xblock-utils==2.1.1
+django-statici18n==1.9.0
+edx-i18n-tools==0.5.3
+Mako==1.1.3
+bleach==3.1.5
+html-xblock~=0.2.0
```

Comment:
@taniwha, till the related `html-xblock` [PR](https://github.com/open-craft/xblock-html/pull/14) is merged, this will cause an error. So can you use the GitHub URL to your PR branch as the requirement here to get the CI checks to pass?